packages/base/tests/functional/basics/ErrorHandling.lua

```diff
@@ -237,6 +237,16 @@ return{
 		unitTest:assertEquals(suggestion("aaaab", t), "aaaaa")
 		unitTest:assertNil(suggestion("ddddd", t))
 	end,
+	suggestionMsg = function(unitTest)
+		local t = {
+			aaaaa = true,
+			bbbbb = true,
+			ccccc = true
+		}
+
+		unitTest:assertEquals(suggestionMsg(suggestion("aaaab", t)), " Do you mean 'aaaaa'?")
+		unitTest:assertEquals(suggestionMsg(nil), "")
```

Comment:
Do not use nil as an argument for a function. Just call it without any argument.